### PR TITLE
New version: LinwoodCloud.Butterfly version 2.1.0-rc.1

### PR DESCRIPTION
**Pull request by CodeDoctorDE**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151636)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335215
> 2024-04-30 05:52:55.210 [CLI ] Package hash verification failed. SHA256 in manifest [73b4af8c8a255ccdc4ce71b32620aaaffc3206ad4b4a3a4432cb9912f3b9b76e] does not match download [8a19f7142d2a69d4e3f60a4aad315db51e2d522874e9b30d67f57cfa9b847d56]
> 2024-04-30 05:52:52.277 [CLI ] Package hash verification failed. SHA256 in manifest [73b4af8c8a255ccdc4ce71b32620aaaffc3206ad4b4a3a4432cb9912f3b9b76e] does not match download [8a19f7142d2a69d4e3f60a4aad315db51e2d522874e9b30d67f57cfa9b847d56]

(Automated response - build 878.)

**Comment by stephengillie** [comment]
This version number has `rc` - is it a release candidate? Should it be added to a `PackageIdentifier` such as `LinwoodCloud.Butterfly.RC` instead?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\Linwood\Butterfly\butterfly.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Programs\Linwood\Butterfly\butterfly.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean Dependency on `Microsoft.VCRedist.2015+.64` (or `.x86`).

(Automated response - build 878.)

**Comment by CodeDoctorDE** [comment]
Yeah sure, we could add it, but I'm not sure how

File: manifests/l/LinwoodCloud/Butterfly/2.1.0-rc.1/LinwoodCloud.Butterfly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: LinwoodCloud.Butterfly
+PackageVersion: 2.1.0-rc.1
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-04-30
+Installers:
```
Comment (stephengillie) [inline manifests/l/LinwoodCloud/Butterfly/2.1.0-rc.1/LinwoodCloud.Butterfly.installer.yaml:12]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
Hi @Manually-Validated,

This package installs and launches normally on a Windows 10 VM.

(Automated response - build 880.)

**Comment by stephengillie** [comment]
Hi @CodeDoctorDE,

> This PR's version number 2.1.0-rc.1 has 4 parameters (sets of numbers between dots - major, minor, etc), which is greater than the current manifest's version 2.0.3, which has 3 parameters.

Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)

(Automated response - build 880)

**Comment by CodeDoctorDE** [comment]
image
Yes this should be the case.

Like I said [here](https://github.com/microsoft/winget-pkgs/pull/151636#issuecomment-2085961061) I don't think it's not a good idea to use a .RC identifier

**Review by stephengillie — APPROVED** [review]

**Pull request merged**